### ISSUE TITLE
Setting "value" of <Toggle> not possible

### ISSUE THREAD

**Issue by LukeOwlclaw** [OfficeDev/office-ui-fabric-react#11347]
### Environment Information

- **Package version(s)**: 7.60.0
- **Browser and OS versions**: Chrome

### Describe the issue:

I want to create a stateful ``<Toggle>`` component which contains the current check state as ``value``. So basically I am doing:

`<Toggle {...otherProps} value={this.state.checked ? "1" : "0"}`

But the problem is, that `value` is not available within `<Toggle>` because the props are defined as:

`export interface IToggleProps extends React.HTMLAttributes<HTMLElement> { ...`

whereas they need to be:

`export interface IToggleProps extends React.ButtonHTMLAttributes<HTMLElement> { ...`

Please correct me if I am missing something!

**Comment by jdhuntington** [comment]
The `Toggle` control exposes the `checked` prop as a boolean, so you should be able to say
```jsx
<Toggle checked={this.state.checked} />
```

I believe the expected use for this control is different than run-of-the-mill form components, which is likely why the name has deviated from a traditional checkbox.

**Comment by LukeOwlclaw** [comment]
Thanks, I noticed that what I wanted to achieve is not possible: I wanted to give the toggle button a value so I can post it via a HTML form. By it seems that `value=` of `<Button>` is not posted. So I guess there is no reason to be able to set it at all. Closing this issue.

For the interested reader: I ended up creating a wrapper for `<Toggle>` which adds a hidden input field which gets posted: `<input type="hidden" name={inputName} value={this.state.checked ? "true" : "false"} />`
